aurora/contracts/src/AuroraErc20FastBridge.sol

```diff
@@ -320,36 +326,53 @@ contract AuroraErc20FastBridge is Initializable, UUPSUpgradeable, AccessControlU
         IEvmErc20 token = tokenInfo.auroraTokenAddress;
         require(tokenInfo.isStorageRegistered == true, "The token storage is not registered");
 
-        require(address(token) != address(0), "The token is not registered");
+        require(address(token) != address(0) ||
```

Comment:
let's add a variable `bool  isNativeTransfer = address(token) != address(0)`

Reply:
fixed: https://github.com/aurora-is-near/fast-bridge-protocol/pull/113/commits/f6bd540da9b60aefd97f5c38e1155c965bb4affa 

https://github.com/aurora-is-near/fast-bridge-protocol/pull/113/commits/3bcebe661a3a0ccf60e139020d774085a4354940